utils/spec-bundle.js

```diff
@@ -62,7 +63,9 @@ Object.assign(global, testing);
  * any file that ends with spec.js and get its path. By passing in true
  * we say do this recursively
  */
-var testContext = require.context(ROOT_DIR, true, /\.spec\.ts/);
+var testContext = skyPagesConfig.runtime.command === 'pact' ?
```

Comment:
I think only your regular expression needs to be changed depending on the command.  No need to duplicate `require.context`.

Reply:
So I tried this again, I also tried setting to regex to a variable, both gave this warning on a test run 
```
WARNING in ../skyux-builder/utils/spec-bundle.js
66:18-25 Critical dependency: require function is used in a way in which dependencies cannot be statically extracted
```
and tests failed.